tests/integration/test_authd/data/config_templates/config_authd_use_password_invalid.yaml

```diff
@@ -0,0 +1,29 @@
+- tags:
```

Comment:
The folder name should be configuration_templates, not config_templates.

Reply:
Applying it in the next commit

Reply:
I will wait for that.